### DISCUSSION TITLE
Faikin options

### DISCUSSION THREAD

**Discussion by sjaakie09** [revk/xESP32-Faikout#366] [Ideas]
Nice tool have 2 units running 
An option would be reset hardware wise
And a way to login to the settings page via home assistant now i cant see it on rental address it is remote site.
And a way to get out kwh usage output to home assistant as it is been shown in the daikin app ?
And is there also a way to get the page https browser does more and more difficult on that?
Thx jd

**Comment by revk** [comment]
Wh usually works.
There is an MQTT command to factory reset. I have pondered if I could make a cunning plan for a hardware reset. Not come up with a plan yet, apart from using USB lead.
